src/setup.py

```diff
@@ -18,6 +18,7 @@
         "pytest>=5.3.5",
         "networkx==2.4",
         "numpy>=1.18.1",
+        "sympy==1.5",
```

Comment:
Sympy is already here, with 1.7 in line 23.

Reply:
I think this pinned version was set because of tensorflow/tensorflow quantum compatibility. Agree that we should stick to the version in dev/master though